diff --git a/lib/v8/src/char-predicates.cc b/lib/v8/src/char-predicates.cc
index 747f419..39aa30e 100644
--- a/lib/v8/src/char-predicates.cc
+++ b/lib/v8/src/char-predicates.cc
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+#define V8_INTL_SUPPORT
 #ifndef V8_INTL_SUPPORT
 #error Internationalization is expected to be enabled.
 #endif  // V8_INTL_SUPPORT
